setup_starport.sh

```diff
@@ -31,6 +31,21 @@ function cleanup {
 
 trap cleanup EXIT
 
+while getopts f o; do
```

Comment:
How does this work? Is this triggered by a `-f` flag? Can that be added as a usage statement somewhere?

Reply:
it's sorta self documenting here
https://github.com/vimc/bb8/pull/26/files#diff-dbf6b7c6b0c7104129d222fc3f07147bR41

do you think that's enough or should I add it somewhere else too?

Reply:
If you could add a general "Usage:" section perhaps, but otherwise that makes sense to me

Reply:
Cool, added at the top of this file